Code/Tools/ProjectManager/Platform/Mac/ProjectBuilderWorker_mac.cpp

```diff
@@ -70,12 +70,13 @@ namespace O3DE::ProjectManager
 
         QString cmakeInstalledPath = cmakeInstalledPathQuery.GetValue();
         QString targetBuildPath = QDir(m_projectInfo.m_path).filePath(ProjectBuildPathPostfix);
-        QString launcherTargetName = m_projectInfo.m_projectName + ".GameLauncher";
+        QString gameLauncherTargetName = m_projectInfo.m_projectName + ".GameLauncher";
+        QString serverLauncherTargetName = m_projectInfo.m_projectName + ".ServerLauncher";
 
         return AZ::Success(QStringList{cmakeInstalledPath,
                                         "--build", targetBuildPath,
                                         "--config", "profile",
-                                        "--target", launcherTargetName, ProjectCMakeBuildTargetEditor});
+                                        "--target", gameLauncherTargetName, serverLauncherTargetName, ProjectCMakeBuildTargetEditor});
```

Comment:
Actually we don't build the ServerLauncher on MacOS as the PAL_TRAIT_BUILD_SERVER_SUPPORTED value is FALSE
https://github.com/o3de/o3de/blob/development/cmake/Platform/Mac/PAL_mac.cmake#L14

Reply:
good looking out! fixed!